opentelemetry/proto/logs/v1/logs.proto

```diff
@@ -127,7 +127,7 @@ message LogRecord {
   // Short event identifier that does not contain varying parts. Name describes
   // what happened (e.g. "ProcessStarted"). Recommended to be no longer than 50
   // characters. Not guaranteed to be unique in any way. [Optional].
-  string name = 4;
+  string name = 4 [deprecated = true];
```

Comment:
Add a message until what time.

Reply:
Added. Since we are in Beta the notice is short.

Reply:
We agreed 3 months for beta, 12 months for stable (at least that is how I remember).

Reply:
```
  # "beta" maturity APIs should not introduce backwards-incompatible changes
  # more than once every three months; and when those changes are introduced, the
  # authors will make a best effort to provide compatibility bridges.
```
This says we cannot break more often than 3 months. It doesn't say anything about deprecation notice period. I can make it 3 months if you think it is important. My understanding is that our formal rules do not make deprecation mandatory at all. But, it is fine if you think we want it to also be 3 months.

Reply:
Let's be nice. June 1.

Reply:
I am not sure about this. We plan to declare logs stable this quarter. I would prefer to remove the field _before_ we declare the logs stable. Otherwise we will be breaking a stable protocol, which is worse.

Reply:
Set to March 15, 2022.